frontend/components/NavigationBar.vue

```diff
@@ -15,10 +15,10 @@
               <a
                 href="#"
                 class="bg-gray-900 text-white px-3 py-2 rounded-md text-sm font-medium"
-                >{{ $t("NavigationBar.Map") }}
-              </a>
+                >{{ $t("NavigationBar.Map") }}</a
+              >
               <a
-                href="#"
+                href="#/addcompany"
```

Comment:
Is it by purpose?

Reply:
If you mean changing `href="#"` to `href="#/addcompany"`, then yes. Previous `href` was temporary and direct to main page, now its direct to form. Name of routes are also temporary, they will be changed with Inertia implementaiton.